spec/stacktrace-spec.js

```diff
@@ -230,7 +230,8 @@ describe('StackTrace', function () {
             });
             waits(100);
             runs(function () {
-                expect(server.requests[0].requestBody).toEqual({stack: stackframes});
+                var expectedResponse = JSON.stringify({stack: stackframes});
+                expect(server.requests[0].requestBody).toEqual(expectedResponse);
```

Comment:
Hm.. I'm guessing the default line length limit is 80. 

Sorry about the annoying Hound stuff; don't worry about this kinda thing. I may just disable it since JSHint is part of the build anyway.